### PR TITLE
New package: libkdegames-18.08.0

### PR DESCRIPTION
**Pull request by 1is7ac3**

### CODE REVIEW

File: common/shlibs [hunk 1]
```diff
@@ -16,6 +16,8 @@
 # PLEASE NOTE: when multiple packages provide the same SONAME, the first
 # one (order top->bottom) is preferred over the next ones.
 #
+libKF5KDEGamesPrivate.so.1 libkdegames-18.08.0_1
```
Comment (maxice8) [inline common/shlibs]: are those used immediately by another package ?

Comment (1is7ac3) [reply common/shlibs]: they are not required immediately

**Comment by maxice8** [comment]
On Sun, Aug 19, 2018 at 09:42:27AM -0700, Osdarwuin Isaac Afanador Quiroz wrote:
> 1is7ac3 commented on this pull request.
> 
> 
> 
> > @@ -16,6 +16,8 @@
>  # PLEASE NOTE: when multiple packages provide the same SONAME, the first
>  # one (order top->bottom) is preferred over the next ones.
>  #
> +libKF5KDEGamesPrivate.so.1 libkdegames-18.08.0_1
> 
> they are not required immediately
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/1923#discussion_r211108369

please do not add them immeediately, add them as a package is introduced that requires it

**Comment by maxice8** [comment]
Create a -devel package with the proper locations, take a look at other KDE libraries

File: common/shlibs [hunk 2]
```diff
@@ -16,6 +16,7 @@
 # PLEASE NOTE: when multiple packages provide the same SONAME, the first
 # one (order top->bottom) is preferred over the next ones.
 #
+libKF5KDEGames.so.7 libkdegames-18.08.0_1
```
Comment (maxice8) [inline common/shlibs]: please add it to the bottom of the file and only if there are packages that require it

File: srcpkgs/libkdegames/template [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+homepage="https://www.kde.org/applications/games/"
+distfiles="${KDE_SITE}/applications/${version}/src/${pkgname}-${version}.tar.xz"
+checksum=e0bca02674c5e961db17e8225affc689c437ab78eabcd620a1e38ee7b1d583ca
+
+libkdegames-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision} ${makedepends}"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/qt5
```
Comment (maxice8) [inline srcpkgs/libkdegames/template]: don't add this path to the -devel package

**Comment by maxice8** [comment]
Please notify when you made the proposed changes

**Pull request merged**